graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/AdditionalType.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Expedia, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.expediagroup.graphql
+
+import kotlin.reflect.KType
+
+/**
+ * Used to represent the additional types to be included in the schema
+ * and that can also be picked up at generation time by including all the
+ * interface implementations that may not be used in the code.
+ */
+data class AdditionalType(
```

Comment:
do we need this wrapper? i.e. I somewhat prefer explicit

```kotlin
internal val additionalTypes: MutableSet<KType> = mutableSetOf()
internal val additionalInputTypes: MutableSet<KType> = mutableSetOf()

// over wrapped
internal val additionalTypes: MutableSet<AdditionalType> = mutableSetOf()
```

Reply:
If we want to keep this as internal we don't need this class to be public. But I made it public with the assumption that we would be combining the arguments in #818

Reply:
Moved the class to internal